client/app/projects/profile/profile.controller.js

```diff
@@ -180,6 +180,24 @@ angular.module('observatory3App')
         });
     };  
 
+    $scope.markPast = function(){
+        $http.put('api/projects/'+$scope.project._id+'/markPast').success(function(){
+            notify("Project marked as past project"); 
+            updateProject(); 
+        }).error(function(){
+            notify("Project not marked as a past project")
```

Comment:
It should probably have error in the message - otherwise it is too easy to just gloss over a message